source/randLIB.c

```diff
@@ -129,19 +129,19 @@ void randLIB_add_seed(uint64_t seed)
 uint8_t randLIB_get_8bit(void)
 {
     uint64_t r = randLIB_get_64bit();
-    return (uint8_t) (r >> 56);
+    return (uint8_t)(r >> 56);
```

Comment:
Really? That seems inconsistent. Does it think it's a function call?

Reply:
Why would you leave a space for this cast?

Reply:
It left one for `(uint64_t) arm_random_seed_get() << 32` above. I'm thought it was set to prefer `(int) a` generally, which I'd extend to `(int) (a + b)`. Don't see any particular reason for brackets to attract each other.

Reply:
Though yes, maybe the space is more about separating the identifier. I can sort of buy that.

Reply:
Good catch ! Another case when astyle is not that intelligent 🙄